### PR TITLE
Hotfix/2.4.5

### PR DESCRIPTION
**Pull request by sven1103**
Many thanks for contributing to QUBE!

**PR Checklist**
Please fill in the appropriate checklist below (delete whatever is not relevant). These are the most common things requested on pull requests (PRs).

- [ ] This comment contains a description of changes (with reason)
- [ ] Referenced issue is linked
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] Documentation in `docs` is updated
- [ ] `CHANGELOG.rst` is updated

**Description of changes**
Please state what you've changed and how it might affect the user.

**Technical details**
Please state any technical details such as limitations, reasons for additional dependencies, benchmarks etc. here.

**Additional context**
Add any other context or screenshots here.

### CODE REVIEW

**Review by Zethson — CHANGES_REQUESTED** [review]
Missing the yaml boolean create fixes.

I am honestly surprised that we ran into that, since in my tests I never had this issue.

**Comment by sven1103** [comment]
I cant find the underlying issue for the boolean write-out. Code looks good. Not sure if this is a bug in the ryaml lib though ...

File: qube/common/load_yaml.py [hunk 1]
```diff
@@ -10,4 +10,8 @@ def load_yaml_file(yaml_file_path: str) -> dict:
     """
     path = Path(yaml_file_path)
     yaml = YAML()
+    yaml.boolean_representation = ['False', 'True']
     return yaml.load(path)
+
```
Comment (Zethson) [inline qube/common/load_yaml.py:15]: :eyes:

Comment (sven1103) [reply qube/common/load_yaml.py]: 👁️

**Review by Zethson — APPROVED** [review]
LGTM

**Review by Zethson — APPROVED** [review]

**Pull request merged**